ykrt/src/mt.rs

```diff
@@ -949,4 +957,41 @@ mod tests {
         mtt.control_point(loc, simple_interp_step, &mut ctx);
         assert_eq!(ctx.run, 2);
     }
+
+    #[test]
```

Comment:
Is it worth adding doc comments to new tests, saying what they test at a glance? Unless it's super-obvious?

Reply:
Yeah, I've just done that since I couldn't remember what I was testing here myself. :wink:

Reply:
Fixed in 7e160bbd13c050265ff1e5a8292f471e209dffd6.